client/src/Components/Search.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+
+const Search = () => {
+  return (
+    <form className="my-4 w-75 mx-auto p-3 bg-secondary text-left text-white">
+      <div className="form-group">
+        <label className="lead w-100">
+          <input
+            name="title"
+            type="text"
+            className="form-control"
+            id="title"
+            // value=""
```

Comment:
**Style**

Remember to remove comments, if you don't use them here in the code